src/festivals/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from .views import NimbleView
```

Comment:
This needs to be a generic view based on the `slug` kwarg from `Festival` model.